sdk/subscription/Microsoft.Azure.Management.Subscription/tests/NetCore/ScenarioTests/SubscriptionTests.ScenarioTests.cs

```diff
@@ -3,23 +3,291 @@
 
 using System.Linq;
 using System.Net;
+using System.Net.Http;
 using Microsoft.Azure.Management.Subscription;
 using Microsoft.Azure.Test;
 using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
 using Microsoft.Rest.TransientFaultHandling;
 using Xunit;
+using Microsoft.Azure.Management.Subscription.Models;
+using System.Collections.Generic;
+using System;
+using System.Threading;
 
 namespace ResourceGroups.Tests
 {
+    [TestCaseOrderer("Orderer.Helper.PriorityOrderer", "ResourceGroups.Tests")]
     public class LiveSubscriptionTests : TestBase
     {
+        private const string subscriptionId = "d17ad3ae-320e-42ff-b5a1-705389c6063a";
+        private readonly int TEN_SECONDS = 10 * 1000;
+
         public SubscriptionClient GetSubscriptionClient(MockContext context, RecordedDelegatingHandler handler)
         {
             handler.IsPassThrough = true;
             var client = this.GetSubscriptionClientWithHandler(context, handler);
             return client;
         }
 
+        [Fact]
+        public void AliasCreateGetDelete()
+        {
+            PutAlias();
+            Thread.Sleep(TEN_SECONDS);
```

Comment:
I remember that the CI has 10s timeout, can we split this tests to three so we don't need to sleep?

Reply:
splited